tests/suites/test_suite_psa_crypto.function

```diff
@@ -6378,7 +6378,7 @@ void signature_size(int type_arg,
 {
     psa_key_type_t type = type_arg;
     psa_algorithm_t alg = alg_arg;
-    size_t actual_size = PSA_SIGN_OUTPUT_SIZE(type, bits, alg);
+    size_t actual_size = PSA_SIGN_OUTPUT_SIZE(type, (size_t) bits, alg);
```

Comment:
Like in https://github.com/Mbed-TLS/mbedtls/pull/7225#discussion_r1132229066, the need for this cast may mean that there's a backward incompatible change here. It's less bad in development than in the LTS branch, but if there's a solution that works well for the LTS branch, we might as well do the same here.